scripts/Makefile

```diff
@@ -0,0 +1,32 @@
+#!make
+MAKEFLAGS += --silent
+
+# This allows us to accept extra arguments (by doing nothing when we get a job that doesn't match, 
+# rather than throwing an error).
+%: 
+    @: 
+
+# $(MAKECMDGOALS) is the list of "targets" spelled out on the command line
+stage: 
+	git clone --quiet https://github.com/madelinezec/test-submodules.git build_scripts
```

Comment:
We need to move this to a proper repository -- either back to workerpool or into a separate repo, although I'm not sure why we would want it to be separate? It has a fairly tight coupling with the docs-workerpool-code. Let's discuss.